SPECS/kubevirt/kubevirt.spec

```diff
@@ -236,6 +236,7 @@ install -p -m 0644 cmd/virt-launcher/qemu.conf %{buildroot}%{_datadir}/kube-virt
 %{_datadir}/kube-virt/virt-handler
 %{_bindir}/virt-handler
 %{_bindir}/virt-chroot
+/virt_launcher.cil
```

Comment:
Is this right? Should this be installed directly under the root folder?

Reply:
@harshitgupta1337 .....

Reply:
Yes. Kubevirt's virt-handler container expects the file to be present at the path `/virt_launcher.cil`.

Reply:
This doesn't seem right for a package to install something directly under `/`. Must the virt-handler look for the file only there? It seems like it might be fixed with a configuration change instead of making a package adjust to a tool.